python/lsst/pipe/base/task.py

```diff
@@ -111,7 +111,7 @@ def __init__(self, config=None, name=None, parentTask=None, log=None):
             if config is None:
                 config = getattr(parentTask.config, name)
             self._taskDict = parentTask._taskDict
-            self.log = Log.getLogger(parentTask.log.getName() + '.' + name)
+            loggerName = parentTask.log.getName() + '.' + name
```

Comment:
Unlike `pex_logging`, the child logs do not have to be chained explicitly to the parent log for the logger hierarchy to work using the new `log`.    Now I see where the confusion comes from.  Ensuring only logger names wouldn't work with `pex_logging` but `log` solves that for us.